### ISSUE TITLE
Train-Validation Loss 

### ISSUE THREAD

**Issue by IsraelAbebe** [moein-shariatnia/OpenAI-CLIP#9]
First, thank you for the blog and the repo 

While training this clip model I observed the validation loss increases and never decreases. Is that supposed to happen?

**Comment by moein-shariatnia** [comment]
> First, thank you for the blog and the repo 
> 
> 
> 
> While training this clip model I observed the validation loss increases and never decreases. Is that supposed to happen?
> 
>

Hi. As the used models are rather small and the dataset limited, I believe the models start overfitting the dataset very soon in the training. I remember a  similar pattern where even 1 epoch was enough for the initial results.
